poutyne/framework/callbacks/logger.py

```diff
@@ -123,11 +123,16 @@ def __init__(self, writer):
         super().__init__(batch_granularity=False)
         self.writer = writer
 
+    def on_batch_end(self, batch, logs):
+        self._on_batch_end_write(batch, logs)
+
     def _on_batch_end_write(self, batch, logs):
         """
         We don't handle tensorboard writing on batch granularity
         """
-        pass
+        for layer_name, layer_gradient in self.model.model.named_parameters():
+            if layer_gradient.requires_grad and ("bias" not in layer_name):
+                self.writer.add_histogram(layer_name, layer_gradient.grad.abs().mean(), batch)
```

Comment:
I believe this is code you forgot to remove.

Reply:
Bad Git :(